src/bots/slack/controller.ts

```diff
@@ -0,0 +1,266 @@
+import { SlackAdapter, SlackBotWorker } from 'botbuilder-adapter-slack';
+import { Botkit, BotkitMessage } from 'botkit';
+import { Response, Request } from 'express';
+import { storage } from './storage';
+import { SLACK_WEBHOOK } from '../../constants';
+// Conversations
+import { onMessageConversation } from './conversations/message.conversation';
+
+export const initSlackController = (adapter: SlackAdapter) => {
+  const controller = new Botkit({
+    adapter,
+    webhook_uri: SLACK_WEBHOOK,
+    storage,
+    // ...other options
+  });
+
+  controller.ready(async () => {
+    // Generic message response
+    onMessageConversation(controller);
+
+    // On DM example
```

Comment:
Could we add a note to remove all these examples? Alternatively, we can extract these out to example files that can be removed.

Reply:
I'm extracting them.  and removing the one we don't want right now